manifests/params.pp

```diff
@@ -76,6 +76,10 @@
   # Custom puppetmaster
   $puppetmaster              = $::puppetmaster
 
+  # Hashes containing additional settings
+  $additional_settings       = {}
```

Comment:
Well, yeah, was thinking the same, except that the master one is called 'server'. Looking at the semantics of the other main options I opted for just additional_settings. But main is fine ...